prism-languages.php

```diff
@@ -5,7 +5,7 @@
  * @return array Hash of lang => language
  */
 function mkaz_code_syntax_block_get_supported_languages() {
-	
+
 	$languages = array(
 		"apacheconf" => "Apache Config",
 		"bash" => "Bash/Shell",
```

Comment:
:no_entry_sign: **Error**: String &quot;bash&quot; does not require double quotes; use single quotes instead (*Squiz.Strings.DoubleQuoteUsage.NotRequired*).